docs/docsite/rst/userguide/jobs.rst

```diff
@@ -14,9 +14,9 @@ The default view is collapsed (**Compact**) with the job name, status, job type,
 |Jobs - home with example job|
 
 .. |Jobs - home with example job| image:: ../common/images/jobs-home-with-example-job.png
-
+                                  :alt: Jobs List with Example Job
```

Comment:
This should really read, "Jobs List with Example Jobs" since there are multiple jobs.